src/ResourceManager/Network/ChangeLog.md

```diff
@@ -16,7 +16,7 @@
     ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
     * Overview of change #1
         - Additional information about change #1
--->
+
```

Comment:
Please add this line back - it is causing everything to be commented out.

Reply:
done